opennms-web-api/src/main/java/org/opennms/web/svclayer/model/RelativeTimePeriod.java

```diff
@@ -48,6 +48,16 @@
     
     static {
         s_defaultPeriods = new RelativeTimePeriod[] {
+                new RelativeTimePeriod("1_hour", "Last Hour", Calendar.HOUR,
```

Comment:
Do you want me to change the existing ones to one line also? Looks like the rest of the file is formatted for max 80 columns.